docs/tutorials/hyp3_isce2_burst_stack_for_ts_analysis.ipynb

```diff
@@ -0,0 +1,471 @@
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "\n",
+    "import asf_search as asf\n",
+    "import pandas as pd\n",
+    "\n",
+    "search_results = asf.search(\n",
+    "        platform=asf.SENTINEL1,\n",
+    "        polarization=asf.VV,\n",
+    "        intersectsWith='POLYGON((-135.7684 57.0473,-135.7389 57.0473,-135.7389 57.0583,-135.7684 57.0583,-135.7684 57.0473))',\n",
+    "        start='2014-05-01',\n",
+    "        end='2015-05-01',\n",
+    "        processingLevel=asf.BURST,\n",
+    "        beamMode=asf.IW,\n",
+    "        flightDirection=asf.DESCENDING,\n",
+    "    )"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "9f7ccdcc",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "\n",
+    "baseline_results = asf.baseline_search.stack_from_product(search_results[-1])\n",
+    "\n",
+    "columns = list(baseline_results[0].properties.keys()) + ['geometry', ]\n",
+    "data = [list(scene.properties.values()) + [scene.geometry, ] for scene in baseline_results]\n",
+    "\n",
+    "stack = pd.DataFrame(data, columns=columns)\n",
+    "stack['startTime'] = stack.startTime.apply(parse_date)\n",
+    "\n",
+    "stack = stack.loc[(stack_start <= stack.startTime) & (stack.startTime <= stack_end)]\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "e8d9f38b",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "\n",
+    "sbas_pairs = set()\n",
+    "\n",
+    "for reference, rt in stack.loc[::-1, ['sceneName', 'temporalBaseline']].itertuples(index=False):\n",
+    "    secondaries = stack.loc[\n",
+    "        (stack.sceneName != reference)\n",
+    "        & (stack.temporalBaseline - rt <= max_temporal_baseline)\n",
+    "        & (stack.temporalBaseline - rt > 0)\n",
+    "    ]\n",
+    "    for secondary in secondaries.sceneName:\n",
+    "        sbas_pairs.add((reference, secondary))\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "9e5c5b0b",
+   "metadata": {},
+   "source": [
+    "## 2. Request On Demand InSAR products from ASF HyP3\n",
+    "\n",
+    "Use your [NASA Earthdata login](https://urs.earthdata.nasa.gov/) to connect to [ASF HyP3](https://hyp3-docs.asf.alaska.edu/)."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "be78f415",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "\n",
+    "import hyp3_sdk as sdk\n",
+    "\n",
+    "hyp3 = sdk.HyP3('https://hyp3-test-api.asf.alaska.edu', prompt=True)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "e1dec3dd",
+   "metadata": {
+    "scrolled": true
+   },
+   "outputs": [],
+   "source": [
+    "\n",
+    "jobs = sdk.Batch()\n",
+    "for reference, secondary in sbas_pairs:\n",
+    "    job = { \"job_parameters\": { \"apply_water_mask\": False, \"granules\": [reference,secondary],\"looks\": \"20x4\" }, \"job_type\": \"INSAR_ISCE_BURST\", \"name\": project_name }\n",
```

Comment:
Can we break the `job = ...` line into multiple lines, like this?
```python3
    job = {
        "job_parameters": {
            "apply_water_mask": False,
            "granules": [reference, secondary],
            "looks": "20x4"
        },
        "job_type": "INSAR_ISCE_BURST",
        "name": project_name
    }
```

Reply:
good idea, will do it.